views/stats.html

```diff
@@ -0,0 +1,84 @@
+% include('base.html', title="User Statistics")
+<body>
+	<div class="container text-center">
+		<div>
+			<h1>User Statistics</h1>
+		</div>
+		<div class="container" style="height:3rem;">
+			<a href="/dashboard" class="btn btn-primary">Dashboard</a>
+			<a href="/logout" class="btn btn-primary">Logout</a>
+		</div>
+		<table class="table">
+			<thead>
+				<tr>
+					<th>Submission History</th>
+				</tr>
+			</thead>
+		</table>
+		<table class="table">
+			<tbody style="height:20rem;display:inline-block;overflow-y:scroll">
+				<tr style="font-weight:bold;">
+					<td style="padding-right:10rem">Contest</td>
+					<td style="padding-right:6rem">Question Number</td>
+					<td style="padding-right:8rem">Time submitted</td>
+					<td style="padding-right:10rem">Correct/Not Correct</td>
+				</tr>
+				% for sub_history in sub_history :
+				<tr>
+					<td style="padding-right:10rem">{{sub_history[0]}}</td>
```

Comment:
```suggestion
					<td style="padding-right:10rem">{{sub_history.code}}</td>
```

Reply:
these kinds of changes make it more readable. Will have to make it a dictionary instead though